src/components/Dropdown/Dropdown.st.css

```diff
@@ -115,14 +241,15 @@
     stroke: none;
 }
 
-.dropdown::popoverContent {
-    border-width: 0;
+.root.overrideStyleParams .dropdown::popoverContent {
+    border-width: 1px;
+    border-color: rgba(0, 0, 0, 0.4);
```

Comment:
Why was the border added here?

Reply:
because of this requirement - "Menu border should be constant: 1px color-5 with 40% opacity". have i done it wrong?

Reply:
Not at all, it's just changing the behaviour we have now.
But I guess that's ok.
But maybe this should be under:
```css
.dropdown::popoverContent
```

and not 
```css
.root.overriderStyleParams .dropdown::popoverContent
```

Meaning, this is a change that should affect the default dropdown design, right?
Not only when overriding the colors